libs/execution/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@
 
 cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
 
-set(parallel_executors_headers
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

Comment:
What is this one for? I don't see any `libs/execution/cmake` directory...

Reply:
This is more for consistency with other modules than anything else.

Reply:
Ah, indeed. Some modules have that line, some don't. I've been trying to remove them since the modules just had empty `cmake` directories, but now it looks like none of them have the directory anymore. I'd rather remove the line in all modules and include it only if needed. What do you think?

Reply:
>  I'd rather remove the line in all modules and include it only if needed. What do you think?

Fair enough!